### DISCUSSION TITLE
functionality of the ftp server function?

### DISCUSSION THREAD

**Discussion by donzefe** [TeamAmaze/AmazeFileManager#4236] [Ideas]
Hi!
I like the app, thank you!

Despite all the advantages the app offers, I am missing one thing: I often use the FTP server function to browse the files on my smartphone with my computer. However, I frequently need to access a file on the SD card in addition to a file from the internal storage. To do so, I always have to stop the FTP server, select the other folder to share, and then reconnect. This is quite cumbersome.

What I would be glad about is the possibility to share both the internal storage and the SD card at the same time.

Perhaps this could be done by having the option to add multiple share folders?

Thank you for your work!
Best regards

**Comment by TranceLove** [comment]
Cool idea - perhaps mount the secondary share folders to specific mount points.

However I expect code will be overly complex, when multiple SAF `content://` URIs are involved. This may not happen without comprehensive integration test suites.

**Reply by donzefe** [reply]
Thanks for your reply!

As I'm not that experienced, I think I cannot help you implement this.

The only possibly helpful thing I know is another app (closed source and with other disadvantages), with which I am able to share both the internal and the SD storage at the same time. It seems to me that the developer does not invest much in updates, which leads me to conclude that it does not have a lot of compatibility problems.

If this could help, I can tell you more about it.
